tooling/strace_analysis.sh

```diff
@@ -22,19 +22,27 @@
 
 # Before executing this script, strace output files need to be generated
 # $1 is path of strace output folder
-# $2 is path of temurin-build folder, for exmaple: /home/user/Documents/temurin-build"
+# $2 is path of temurin-build folder, for example: /home/user/Documents/temurin-build"
 # $3 is javaHome
 # $4 is classpath
 # $5 is sbomJson
+# $6 is path of openjdk build output folder
+# $7 is path of cloned openjdk folder
+# $8 is Optional, path of devkit
 
-set +eu
+set -eu
```

Comment:
Was the deliberate?

Reply:
@karianna yes see https://github.com/adoptium/temurin-build/issues/3689, "set +eu" meant it would swallow failures!, rather than check for successful shell command completion. set -eu means we properly manage failures